app/Models/Vehicle.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Vehicle extends Model
+{
+    protected $table = 'vehicles';
+
+    protected $primaryKey = 'vehicle_id';
+
+    public $timestamps = false;
+
+    protected $fiillable = [
+        'vehicle_name',
+        'rent_expense'
+    ];
+
+    protected $attributes = [
+        'license_plate' => 'Cập nhật sau',
+        'description'   => 'Bình thường',
+        'image'         => 'Cập nhật sau'
+    ];
+
+    public function vehicleType()
+    {
+        return $this->belongsTo('App\Models\VehicleType', 'vehicle_type_id', 'vehicle_type_id');
```

Comment:
return $this->belongsTo(VehicleType::class, 'vehicle_type_id', 'vehicle_type_id');